test/vm_tests/builder/declaration_parsing_test.dart

```diff
@@ -390,6 +390,29 @@ main() {
               expect(declarations.state.meta.keyNamespace, 'baz');
               expect(declarations.component.meta.isWrapper, isTrue);
             });
+            test('a stateful Component2', () {
```

Comment:
#nit could use a newline before this
```suggestion

            test('a stateful Component2', () {
```